### ISSUE TITLE
Screen reader describes OSD controls as clickable divs

### ISSUE THREAD

**Issue by jamesmisson** [UniversalViewer/universalviewer#1170]
UV version: universalviewer@4.0.25

**I'm submitting a:**

- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

This issue has been split out from here: https://github.com/UniversalViewer/universalviewer/issues/1085
When using a screen reader, the OpenSeaDragon controls (zoom, rotate, and the image adjustment and home buttons currently in dev) are described as 'zoom clickable', and should be just 'zoom'. It would be better for accessibility to have them as button elements.
See this comment: https://github.com/UniversalViewer/universalviewer/issues/1085#issuecomment-2413948841

**Expected behavior:**

Screen readers describe them as buttons with their title.

**Steps to reproduce:**

1. activate NVDA screenreader (requires Windows)
2. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
3. use the keyboard to navigate to the items mentioned and observe how they are announced by the screenreader.

**Other information:**

These buttons are generated by OSD and div is the default element. It's possible to define which element they are generated as via the Button class. See OSD docs here: https://openseadragon.github.io/docs/OpenSeadragon.Button.html

**Comment by yslflow** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by LlGC-jop** [comment]
Line 125 in https://github.com/openseadragon/openseadragon/blob/master/src/button.js looks like it either can have a pre-existing html element passed when creating a button, or will use a div.

So I don't think there'll be a way to easily change the element type on our end, unless we do it at runtime?

**Comment by LlGC-jop** [comment]
It's a bit clunky (my jQuery is rusty) but something like this should work for the zoom in button etc.

```
let $oldZoomIn = this.$viewer.find('div[title="Zoom in"]');
this.$zoomInButton = $("<button />").append($oldZoomIn.contents());
this.$zoomInButton.insertAfter($oldZoomIn);
$oldZoomIn.remove();
this.$zoomInButton.attr("tabindex", 0);
this.$zoomInButton.attr("title", this.content.zoomIn);
this.$zoomInButton.attr("aria-label", this.content.zoomIn);
this.$zoomInButton.addClass("zoomIn viewportNavButton");
```
